x11/windowmaker/Portfile

```
@@ -1,8 +1,10 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem 1.0
 
 name		windowmaker
-version		0.95.3
-revision        2
+version		0.95.9
+revision        1
```

Since this software is now hosted at GitHub, it would be a good idea to use the github portgroup. Besides simplifying the port, this should also fix the livecheck.

```suggestion
PortGroup	github 1.0

github.setup	wmaker windowmaker 0.95.9 wmaker-
revision        0
```